RoslynSecurityGuard.Vsix/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.Identity.Core" version="2.2.1" targetFramework="net452" />
```

Comment:
It is possible to avoid including Identity Core. It seems like only the test cases require it.
This will bundle the Identity Core DLL in the VSIX package.